libsolidity/codegen/ContractCompiler.cpp

```diff
@@ -586,7 +586,6 @@ void ContractCompiler::initializeStateVariables(ContractDefinition const& _contr
 	solAssert(!_contract.isLibrary(), "Tried to initialize state variables of library.");
 	for (VariableDeclaration const* variable: _contract.stateVariables())
 	{
-		solUnimplementedAssert(variable->referenceLocation() != VariableDeclaration::Location::Transient, "Transient storage variables not supported.");
```

Comment:
For transient state variable assert again that ``variable->value()`` is empty and then just do nothing.

Reply:
Done.